src/tiles/clickable-tile.component.ts

```diff
@@ -0,0 +1,70 @@
+import {
+	Component,
+	Input
+} from "@angular/core";
+
+/**
+ * Build application's clickable tiles using this component.
+ *
+ * ## Basic usage
+ *
+ * ```html
+ * <ibm-clickable-tile>
+ * 		tile content
+ * </ibm-clickable-tile>
+ * ```
+ *
+ * @export
+ * @class ClickableTile
+ * @implements {OnInit}
+ */
+@Component({
+	selector: "ibm-clickable-tile",
+	template: `
+	<a
+		[ngClass]="{
+			'bx--link': true,
```

Comment:
We normally provide only classes that are conditionally assigned in `ngClass` and other in "regular" class. It kinda says: these classes are programmatically updated.

So you'd have

```html
class="bx--link bx--tile bx--tile--clickable"
[ngClass]="{
	'bx--tile--is-clicked': clicked
}"
```

or something along those lines.